assets/sass/_header.scss

```diff
@@ -95,7 +95,7 @@ Header
         display: none;
       }
 
-      @media screen and (max-width: $width-phone) {
+      @include max-width($width-phone) {
```

Comment:
Expected item on line 98 to appear before line 94. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets